### PR TITLE
feature(README): Users should be able to run the projects

### LINKED ISSUE #7: README
README with info and code bases.

### PR DESCRIPTION
**Pull request by michaelachrisco**
## Changes:
1. README.md contains presentation specific information.

 ## Purpose
1. Users should be able to access the presentation.
2. Users should be able to run all sample projects.

Closes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+and modified to suit our needs.
+
+to run: `deno run --allow-net examples/oak-todo/server.ts`
+
+### WebAssembly
+Example from Webassembly was based on prevous talks I gave on the subject here: https://github.com/michaelachrisco/ToyLisp
+I am not including this code as its used as a reference on what webassembly can do for the web.
+
+### Deno Desktop App
+Example web based decktop app (kind of like electron) using Rust and Deno.
```
Comment (ryekerjh) [inline README.md]: desktop**

**Review by ryekerjh — COMMENTED** [review]
LGTM, just one typo that I found. I haven't tried to run the projects. Are we going to have all the devs try to run this on thier local?

**Comment by michaelachrisco** [comment]
> LGTM, just one typo that I found. I haven't tried to run the projects. Are we going to have all the devs try to run this on thier local?

Thanks!

I wasn't planning on it during the talk, although if they have a VM/docker of Ubuntu the code will work. I guess I could say: "If you happen to have Ubuntu running on your system somewhere you can follow along"? What do you think?

Some of these projects may have dependencies on the Mac OS and Windows side, and this has only been tested on Ubuntu. Deno 3rd party libraries seem to be picky ATM. I think this would need to be tested and documented before going that direction.

File: README.md (hunk 1 above)
Comment (michaelachrisco) [reply README.md]: Fixed via: 17c63f050460bb9ce853349670f2bf131132449d

Typos are the worst!

**Pull request merged**